src/components/v5/common/ActionSidebar/partials/MultiSigSidebar/partials/VoteButton/VoteButton.tsx

```diff
@@ -10,23 +11,29 @@ import { extractColonyRoles } from '~utils/colonyRoles.ts';
 import { extractColonyDomains } from '~utils/domains.ts';
 import { getMultiSigRequiredRole } from '~utils/multiSig.ts';
 import Button from '~v5/shared/Button/Button.tsx';
+import TxButton from '~v5/shared/Button/TxButton.tsx';
+import { type ButtonProps } from '~v5/shared/Button/types.ts';
 
 const displayName =
   'v5.common.ActionSidebar.partials.MultiSig.partials.VoteButton';
 
-interface VoteButtonProps {
+interface VoteButtonProps extends ButtonProps {
   actionType: ColonyActionType;
   multiSigId: string;
   multiSigDomainId: number;
   voteType: Exclude<MultiSigVote, MultiSigVote.None>;
+  setIsButtonPending: (isButtonPending: boolean) => void;
 }
 
 const VoteButton: FC<VoteButtonProps> = ({
   actionType,
   multiSigId,
   multiSigDomainId,
   voteType,
+  setIsButtonPending,
+  ...buttonProps
 }) => {
+  const [isPending, setIsPending] = useState(false);
```

Comment:
what if we control this from the parent component? if that component cares about voting pending, we can just send the `isButtonPending` with other props too